lib/Validator.js

```diff
@@ -324,9 +324,6 @@ class Validator {
             ) {
               return 'Value does not match pattern';
             }
-
-            // Here slotType value tested and is ok!
-            continue;
```

Comment:
With `continue` here, the `anyOf`, `allOf`, etc. processing code further down inside this `for` block was not being reached.